src/components/Match/renderMatch.js

```diff
@@ -160,19 +160,20 @@ function generateWardLog(match) {
       })
     ;
 
-    const observers = extractWardLog('observer', safePlayer.obs_log, safePlayer.obs_left_log);
-    const sentries = extractWardLog('sentry', safePlayer.sen_log, safePlayer.sen_left_log);
+    const observers = extractVisionLog('observer', safePlayer.obs_log, safePlayer.obs_left_log);
+    const sentries = extractVisionLog('sentry', safePlayer.sen_log, safePlayer.sen_left_log);
     return _.concat(observers, sentries);
```

Comment:
prefer native method here (observers.concat(sentries) as it also doesn't mutate, just copies.